sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -72,6 +182,45 @@ export interface OperationOptions {
     tracingOptions?: OperationTracingOptions;
 }
 
+// @public
+export interface QueueDescription {
+    authorizationRules?: AuthorizationRule[];
+    autoDeleteOnIdle?: string;
+    deadLetteringOnMessageExpiration?: boolean;
+    defaultMessageTtl?: string;
+    duplicateDetectionHistoryTimeWindow?: string;
+    enableBatchedOperations?: boolean;
+    enablePartitioning?: boolean;
+    forwardDeadLetteredMessagesTo?: string;
+    forwardTo?: string;
+    lockDuration?: string;
+    maxDeliveryCount?: number;
+    maxSizeInMegabytes?: number;
+    // (undocumented)
```

Comment:
Is this undocumented because you plan to remove it once the runtime info methods are added?

Reply:
Yes, there are helper methods for the tests that rely on this, I've moved this to the runtimeinfo APIs in the second PR.